src/common/i18n/locales/en-US.json

```diff
@@ -211,7 +211,9 @@
     "vote-error": "Previous value is not acceptable. Vote with a different value.",
     "vote-remove": "Voting 0% unvotes the content.",
     "vote-warning": "Changing vote...",
-    "no-replies": "No responses yet. Click to respond."
+    "no-replies": "No responses yet. Click to respond.",
+    "old-post-error": "This post is older than 7 days. Voting does not give any credit to the author.",
```

Comment:
Change text to this:
`Voting on Content after their payout does not generate any new rewards.`